.github/workflows/e2e-test-manual.yml

```diff
@@ -269,16 +244,10 @@ jobs:
           kubeconfig: ${{ steps.e2e_test.outputs.kubeconfig }}
 
       - name: Always delete IAM configuration
-        if: always() && inputs.test == 'iamcreate' && inputs.cloudProvider != 'azure' # skip for Azure, as the SP / MI does not have the required permissions
+        if: always()
+        continue-on-error: true
```

Comment:
I think this is needed for `name: Always terminate cluster` above but not strictly for this, since it is most likely the last step, right?

Reply:
Yes, but my main point was that it is missing for the `name: Always terminate cluster` step above, right?

Reply:
I think this is due to the fact that a failure on cluster termination needs to be resolved manually?

Reply:
After a discussion with @katexochen, I think that we are using the `continue-on-error` in too many cases. Since we want to know if this deletion fails, I'd go with removing is here.